helsingborg-stad/app-mitt-helsingborg
# feat: add calendar screen and navigation

## Explain the changes you’ve made

I have added a Calendar screen with its own stack navigator and updated the bottom bar navigator to include it. The tab is conditionally rendered only in dev mode so it will not be visible in production until the check is removed.
Since the backend for retrieving the data is not done yet, I have created mock booking data which is returned with a slight delay in order to simulate retrieving data from a backend. 

## Explain why these changes are made

This screen is part of the booking flow and allows users to see their existing bookings.

## Explain your solution

The solution is based on the `CasesOverview` screen.  A problem with `TabNavigator` not finding the correct color to display was solved by changing the choice of color from depending on the index of the tab to depending on path name instead.

## How to test the changes?

1. Checkout this branch
2. Make sure application is in dev mode.
3. Fire upp the simulator by running the command `yarn ios`
4. The new tab can be found at the bottom of the screen.

## Was this feature tested in the following environments?
- [] Storybook on a iOS device/simulator.
